homeassistant/components/logbook/__init__.py

```diff
@@ -127,7 +127,7 @@ def log_message(service: ServiceCall) -> None:
     possible_merged_entities_filter = convert_include_exclude_filter(merged_filter)
     if not possible_merged_entities_filter.empty_filter:
         filters = sqlalchemy_filter_from_include_exclude_conf(merged_filter)
-        entities_filter = possible_merged_entities_filter
+        entities_filter = possible_merged_entities_filter.get_filter()
```

Comment:
We don't need the wrapped object here since we don't care if entities are specifically included or excluded so we can use the underlying filter which does not have the `__call__` overhead